test/object-store/util/baas_admin_api.cpp

```diff
@@ -873,6 +873,9 @@ AppSession create_app(const AppCreateConfig& config)
         auto queryable_fields = nlohmann::json::array();
         const auto& queryable_fields_src = config.flx_sync_config->queryable_fields;
         std::copy(queryable_fields_src.begin(), queryable_fields_src.end(), std::back_inserter(queryable_fields));
+        auto asymmetric_tables = nlohmann::json::array();
+        const auto& asymmetric_tables_src = config.flx_sync_config->asymmetric_tables;
+        std::copy(asymmetric_tables_src.begin(), asymmetric_tables_src.end(), std::back_inserter(asymmetric_tables));
```

Comment:
this is really minor, but if you have the schema, you could also just loop through the tables in the schema and build your list of asymmetric tables from that rather than requiring a separate vector of table names in the app config.